src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -403,6 +409,10 @@ class MediaBaseViewer extends BaseViewer {
             this.debouncedEmit('volume', volume);
             this.mediaEl.volume = volume;
         }
+
+        if (this.controls) {
```

Comment:
Is this guard clause necessary? There's another one inside `renderUI`.

Reply:
It's not strictly necessary since `renderUI` is a noop otherwise. I can remove it.

Reply:
It's ok. I just find it's not consistent with line 553: https://github.com/box/box-content-preview/pull/1302/files#diff-242d253d5790db64a7f3a5f5a65fe30ee9045543c45bd03b46c77271058002d0R553. It doesn't have the guard clause.